src/pages/guides/using/index.md

```diff
@@ -17,4 +17,5 @@ Setting up an integration to subscribe to Adobe I/O Events is straightforward. T
 * [Imaging APIs](https://developer.adobe.com/firefly-services/docs/photoshop/features/#webhooks-through-adobe-io-events)
 * [InDesign APIs](indesign-apis/indesign-apis-events-data-stream-setup.md)
 * [Marketo Data Streams](marketo/marketo-data-streams.md)
-* [Your own Custom Events Provider](custom-events.md)
+* [Content Log Events](contentlogs/contentlogs-setup.md)
```

Comment:
Trying to keep this list in alphabetical order, so can you please add this after line 16.

Also, you should add this doc in the following places as well:
1. [FAQ page](https://github.com/AdobeDocs/adobe-io-events/blob/main/src/pages/support/faq.md?plain=1#L25)
2. [Sidebar Navigation](https://github.com/AdobeDocs/adobe-io-events/blob/main/src/pages/config.md?plain=1#L37)

Please try and preserve the alphabetical order at every place.